app/models/event.rb

```diff
@@ -14,4 +14,8 @@ def rated_by?(user)
   def average_rating
     ratings.pluck(:rating_score).inject(0, :+) / (ratings.count.nonzero? || 1)
   end
+
+  def rating_for(user)
+    ratings.find_by(user_id: user.id)
+  end
```

Comment:
@ianlenehan this will already be scoped by a single event. 

it would work the same if you called it in your console like `Event.last.users.where(user_id: 123).first`.

Might be worth trying it out in the console if you got some fake data, just to see if it does what I think it does